tck/tck-runtime/src/main/java/servlet/tck/common/util/ServletTestUtil.java

```diff
@@ -43,6 +43,8 @@ public class ServletTestUtil {
 
   private static Logger LOGGER = LoggerFactory.getLogger(ServletTestUtil.class);
 
+  public static boolean SUPPORT_CROSS_CONTEXT = Boolean.parseBoolean(System.getProperty("servlet.tck.support.crossContext", "true"));
```

Comment:
Should this default to false?

Reply:
I don't have strong opinion. the idea was to be backward compact. (tests running per default)

Reply:
If it defaults to false, an implementation running the the TCK that supports cross-context could easily forget to set this leading to potentially incorrect TCK results.
If it defaults to true, an implementation running the the TCK that does not support cross-context will see test failures if they forget to set this.
I think defaulting to true is the safer option.